NewDawn/NewDawn/MainPageViewController.swift

```diff
@@ -16,7 +16,11 @@ class MainPageViewController: UIViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        var user_profiles:[UserProfile] = UserProfileBuilder.getUserProfileListFromLocalStorage()
+//        var user_profiles:[UserProfile] = UserProfileBuilder.getUserProfileListFromLocalStorage()
+        var user_profiles = [
+            UserProfile(data: USER_DUMMY_DATA[0]),
+            UserProfile(data: USER_DUMMY_DATA[1])
+        ]
```

Comment:
This is for test only. @duckmoll Feel free to remove it when you finishes the work for fetching everything from backend